ic-http/src/http_request.rs

```diff
@@ -34,5 +39,14 @@ pub async fn http_request_with_cycles(
     arg: CanisterHttpRequestArgument,
     cycles: u128,
 ) -> CallResult<(HttpResponse,)> {
-    ic_cdk::api::management_canister::http_request::http_request_with_cycles(arg, cycles).await
+    // NOTE: removing `http_request_with_cycles` is a workaround for the dependency issue
+    // when a recent `ic_cdk` version lost the `http_request_with_cycles` function.
+    // TODO: put back `http_request_with_cycles` as soon as it's available in the corresponding CDK version.
```

Comment:
Not sure it's really worth leaving a TODO. The current version looks clean enough to me that I would actually keep it forever, the wrapper hardly adds value imo.